spec/models/assignment_repo_spec.rb

```diff
@@ -53,6 +53,12 @@
 
       expect(repo.valid?).to be_truthy
     end
+    it "expects github_team_id to be nil" do
```

Comment:
Can we add a newline above here? To be consistent with other code in this app, blocks should not touch each other.